test/metabase/test/data/generic_sql.clj

```diff
@@ -248,15 +248,19 @@
 (def load-data-one-at-a-time-parallel! "Insert rows one at a time, in parallel."              (make-load-data-fn load-data-add-ids (partial load-data-one-at-a-time pmap)))
 ;; ^ the parallel versions aren't neccesarily faster than the sequential versions for all drivers so make sure to do some profiling in order to pick the appropriate implementation
 
-(defn default-execute-sql! [driver context dbdef sql]
+(defn- jdbc-execute! [db-spec sql]
+  (jdbc/execute! db-spec [sql] {:transaction? false, :multi? true}))
+
+(defn default-execute-sql! [driver context dbdef sql & {:keys [execute]
```

Comment:
this param should probably be called `execute!` to make it clear that it's a function with side-effects 

https://github.com/bbatsov/clojure-style-guide#changing-state-fns-with-exclamation-mark